presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/rtree/HasExtent.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geospatial.rtree;
+
+import com.facebook.presto.geospatial.Rectangle;
+
+public interface HasExtent
+{
+    Rectangle getExtent();
+    long getSizeInBytes();
```

Comment:
nit: I prefer `getEstimatedSize` or `getEstimatedSizeInBytes` because often we can't know exactly how much memory an object is using and return an estimate or approximation.